pkgs/applications/maui/fetch.sh

```diff
@@ -0,0 +1 @@
+WGET_ARGS=( https://download.kde.org/stable/maui/ -A '*-2.1.2.tar.xz' )
```

Comment:
Someone with a Telegram account could ask upstream whether they publishing under `https://download.kde.org/stable/maui/2.1.2` like KDE Gear and Plasma Mobile Gear.